src/BlazorUI/Bit.BlazorUI/Components/Inputs/Dropdown/BitDropdown.scss

```
@@ -150,7 +161,10 @@
     user-select: none;
     color: $clr-fg-sec;
     position: relative;
-    min-height: spacing(4);
+    // Centers the content of Prefix/SuffixTemplate; the component's own children opt back
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)